integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/iceberg/SparkIcebergCatalogIT.java

```diff
@@ -24,6 +29,13 @@
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class SparkIcebergCatalogIT extends SparkCommonIT {
 
+  private List<SparkTableInfo.SparkColumnInfo> getIcebergSimpleTableColumn() {
```

Comment:
please add the private method to the bottom of the class file

Reply:
updated.